bigint_impl.h typedefs uint16_t and uint32_t but never includes the
header where they are defined. On a standard gnu/linux system that
means you need to #include <stdint.h>. This error occurs whenever
trying to use the API, eg. when compiling curl --with-axtls.

http://sourceforge.net/tracker/?func=detail&aid=3515628&group_id=171217&atid=857113

diff -Naur axTLS.orig//crypto/bigint_impl.h axTLS/crypto/bigint_impl.h
--- axTLS.orig//crypto/bigint_impl.h	2011-01-02 02:45:25.000000000 -0500
+++ axTLS/crypto/bigint_impl.h	2012-04-07 17:36:36.000000000 -0400
@@ -31,6 +31,8 @@
 #ifndef BIGINT_IMPL_HEADER
 #define BIGINT_IMPL_HEADER
 
+#include <stdint.h>
+
 /* Maintain a number of precomputed variables when doing reduction */
 #define BIGINT_M_OFFSET     0    /**< Normal modulo offset. */
 #ifdef CONFIG_BIGINT_CRT
